### DISCUSSION TITLE
Avaid 404 error on Refresh for a not generated page

### DISCUSSION THREAD

**Discussion by bekreativ** [nuxt/nuxt#17569] [Questions]
Hello everybody,

In my application I have a search result page. In /pages/search/[term].vue

When I generate my application with "npm run generate", this page is not generated, and I think it's logic because the "term" could be anything.

In a basic journey, the user have an input text and the text is used like : 
`navigateTo({
                    path: '/search/' + term
                })`

But I have an issue, if I refresh the page (command+R or F5), I get a 404 error.

Is there a tips to avoid the 404 error ?

Thanks and have a nice day.

**Comment by TheAlexLichter** [comment]
In Nuxt 2, there was a [fallback](https://nuxtjs.org/docs/configuration-glossary/configuration-generate#fallback) page that will render all non-generated pages as SPA. In Nuxt 3, Nitro has no equivalent for it yet. Follow #5003 for it ☺️

**Reply by bekreativ** [reply]
Ok thanks you !
